mmdet/models/detectors/single_stage.py

```diff
@@ -160,7 +160,23 @@ def onnx_export(self, img, img_metas):
         # `CornerNet` and `CentripetalNet`, which 'pad_shape' is used
         # for inference
         img_metas[0]['pad_shape_for_onnx'] = img_shape
-        # TODO:move all onnx related code in bbox_head to onnx_export function
-        det_bboxes, det_labels = self.bbox_head.get_bboxes(*outs, img_metas)
+
+        if len(outs) == 2:
+            # add dummy score_facotr
+            outs = (*outs, None)
+
+        if hasattr(self.bbox_head, 'onnx_export'):
+            det_bboxes, det_labels = self.bbox_head.onnx_export(
+                *outs, img_metas)
+        else:
+            warnings.warn(
+                f'{self.__class__.__name__}  may not be'
```

Comment:
rephrase.

Reply:
We need to discuss how to judge we should call  `get_bboxes` instead of `onnx_export` @RunningLeon